plugins/Jobs.cpp

```diff
@@ -161,13 +203,43 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
     
     // ----------------------------------------------------------------------
     else if (SIEquals(request.methodLine, "CreateJob")) {
+        if (request.isSet("repeat") && request.isSet("retryAfter")) {
```

Comment:
Please add a comment above this explaining why auto-retrying jobs aren't supported.  Basically, the pattern is:

```
// An English comment of the code below
someLinesOfCode();

// Note the blank line between the last line of the previous block, and the comment of the next block
```